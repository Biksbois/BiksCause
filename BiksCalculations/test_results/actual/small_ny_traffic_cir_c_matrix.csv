,Clouds,Clear,Mist,Snow,Haze,Rain,Drizzle,Fog,Thunderstorm,overcast clouds,sky is clear,mist,broken clouds,heavy snow,few clouds,scattered clouds,haze,light rain,light intensity drizzle,fog,moderate rain,snow,drizzle,proximity thunderstorm,heavy intensity rain,thunderstorm with light rain,thunderstorm with heavy rain,proximity thunderstorm with rain,heavy intensity drizzle,proximity shower rain,freezing rain,shower snow
Clouds,8.42,0.65,0.15,0.32,0.29,0.17,0.14,0.03,0.19,2.45,0.65,0.15,2.53,0.32,1.6,2.03,0.29,0.17,0.14,0.03,0.1,0.03,0.09,0.28,0.11,0.0,0.0,0.0,0.0,1.18,0.0,0.0
Clear,0.46,18.65,0.04,0.11,0.15,0.06,0.02,0.0,0.08,0.17,18.65,0.04,0.39,0.11,1.88,0.88,0.15,0.07,0.02,0.0,0.06,0.0,0.0,0.11,0.0,0.0,0.0,0.0,0.0,1.4,0.0,0.0
Mist,0.19,0.1,12.58,2.03,2.74,2.12,2.22,2.84,1.75,0.34,0.1,12.58,0.15,2.02,0.21,0.22,2.74,2.13,2.17,2.84,1.93,2.15,2.05,1.85,1.65,1.53,2.02,1.09,2.19,0.0,1.31,3.28
Snow,0.37,0.08,3.01,23.58,1.51,0.96,0.32,2.71,0.14,0.74,0.08,3.01,0.31,23.46,0.12,0.13,1.51,0.93,0.28,2.71,0.35,6.25,0.34,0.21,0.24,0.0,0.0,0.0,0.0,0.0,5.28,0.0
Haze,0.32,0.23,4.59,1.22,13.8,0.46,0.59,1.3,0.28,0.49,0.23,4.59,0.3,1.23,0.27,0.29,13.8,0.4,0.39,1.3,0.22,0.1,0.41,0.4,0.0,0.46,0.0,0.0,6.76,0.0,0.0,10.07
Rain,0.32,0.14,5.43,0.64,0.81,27.15,16.23,0.86,7.85,0.78,0.14,5.43,0.25,0.56,0.0,0.06,0.81,24.07,15.18,0.86,13.47,0.92,10.19,6.05,11.77,8.67,6.42,15.64,4.6,5.16,0.0,0.0
Drizzle,0.31,0.12,9.24,0.16,0.62,22.19,30.02,2.68,4.03,0.81,0.12,9.24,0.09,0.16,0.0,0.05,0.62,20.91,25.35,2.68,9.29,0.34,12.73,2.83,4.05,6.78,8.16,1.94,3.88,0.0,0.0,0.0
Fog,0.06,0.07,28.14,1.89,1.2,1.85,1.78,28.21,0.74,0.13,0.07,28.14,0.04,1.91,0.08,0.0,1.2,1.5,1.81,28.21,1.02,7.84,1.38,1.08,0.0,0.0,0.0,0.0,0.0,0.0,16.63,0.0
Thunderstorm,0.11,0.0,6.31,0.27,9.68,47.56,12.25,0.88,97.76,0.3,0.0,6.31,0.0,0.28,0.0,0.0,9.68,31.33,13.45,0.88,5.54,1.49,1.25,87.98,15.02,80.68,51.14,17.3,0.0,0.0,0.0,0.0
overcast clouds,10.31,0.23,0.22,0.44,0.25,0.28,0.22,0.04,0.41,9.63,0.23,0.22,1.53,0.44,0.32,0.52,0.25,0.27,0.22,0.04,0.18,0.07,0.16,0.6,0.11,0.0,0.0,0.0,0.0,0.0,0.0,0.0
sky is clear,0.46,18.65,0.04,0.11,0.15,0.06,0.02,0.0,0.08,0.17,18.65,0.04,0.39,0.11,1.88,0.88,0.15,0.07,0.02,0.0,0.06,0.0,0.0,0.11,0.0,0.0,0.0,0.0,0.0,1.4,0.0,0.0
mist,0.19,0.1,12.58,2.03,2.74,2.12,2.22,2.84,1.75,0.34,0.1,12.58,0.15,2.02,0.21,0.22,2.74,2.13,2.17,2.84,1.93,2.15,2.05,1.85,1.65,1.53,2.02,1.09,2.19,0.0,1.31,3.28
broken clouds,9.75,0.69,0.11,0.21,0.39,0.15,0.1,0.0,0.0,1.23,0.69,0.11,7.05,0.21,1.32,3.24,0.39,0.16,0.11,0.0,0.07,0.0,0.04,0.0,0.21,0.0,0.0,0.0,0.0,4.76,0.0,0.0
heavy snow,0.4,0.08,2.65,21.57,1.64,0.87,0.33,2.02,0.16,0.78,0.08,2.65,0.35,21.45,0.14,0.15,1.64,0.88,0.29,2.02,0.34,4.55,0.33,0.23,0.26,0.0,0.0,0.0,0.0,0.0,2.33,0.0
few clouds,4.72,2.21,0.1,0.23,0.36,0.0,0.0,0.09,0.0,0.34,2.21,0.1,1.38,0.24,9.85,3.26,0.36,0.0,0.0,0.09,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
scattered clouds,7.63,1.07,0.06,0.2,0.18,0.05,0.06,0.0,0.0,0.67,1.07,0.06,3.39,0.21,2.88,6.89,0.18,0.05,0.07,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
haze,0.32,0.23,4.59,1.22,13.8,0.46,0.59,1.3,0.28,0.49,0.23,4.59,0.3,1.23,0.27,0.29,13.8,0.4,0.39,1.3,0.22,0.1,0.41,0.4,0.0,0.46,0.0,0.0,6.76,0.0,0.0,10.07
light rain,0.36,0.13,4.83,0.75,0.72,23.36,14.55,0.93,5.39,0.84,0.13,4.83,0.27,0.66,0.0,0.09,0.72,20.45,13.23,0.93,10.36,1.12,9.67,4.66,8.87,6.14,6.57,8.0,6.3,7.06,0.0,0.0
light intensity drizzle,0.35,0.13,7.96,0.13,0.74,16.76,26.42,3.02,4.28,0.9,0.13,7.96,0.08,0.13,0.0,0.08,0.74,16.11,22.6,3.02,8.23,0.24,10.45,4.04,3.41,7.2,7.67,2.74,0.0,0.0,0.0,0.0
fog,0.06,0.07,28.14,1.89,1.2,1.85,1.78,28.21,0.74,0.13,0.07,28.14,0.04,1.91,0.08,0.0,1.2,1.5,1.81,28.21,1.02,7.84,1.38,1.08,0.0,0.0,0.0,0.0,0.0,0.0,16.63,0.0
moderate rain,0.25,0.18,8.12,0.33,0.95,53.9,21.98,0.78,11.17,0.67,0.18,8.12,0.15,0.25,0.0,0.0,0.95,47.96,22.15,0.78,28.65,0.43,11.67,7.47,18.39,11.08,0.0,26.61,0.0,0.0,0.0,0.0
snow,0.18,0.0,15.52,94.57,0.48,2.05,0.19,14.74,0.0,0.49,0.0,15.52,0.0,95.58,0.0,0.0,0.48,1.45,0.21,14.74,0.41,43.27,0.45,0.0,0.0,0.0,0.0,0.0,0.0,0.0,35.64,0.0
drizzle,0.24,0.08,13.97,0.23,0.17,80.32,50.03,1.96,3.51,0.64,0.08,13.97,0.1,0.23,0.0,0.0,0.17,62.11,38.84,1.96,12.76,0.6,21.0,0.0,5.78,5.91,9.59,0.0,13.86,0.0,0.0,0.0
proximity thunderstorm,0.24,0.0,6.79,0.55,8.47,22.19,4.29,1.76,54.74,0.65,0.0,6.79,0.0,0.56,0.0,0.0,8.47,13.43,4.71,1.76,9.0,2.98,0.0,49.49,14.08,57.63,0.0,34.59,0.0,0.0,0.0,0.0
heavy intensity rain,0.0,0.0,9.7,0.0,1.35,0,0,0.0,136.86,0.0,0.0,9.7,0.0,0.0,0.0,0.0,1.35,0,0,0.0,36.02,0.0,20.0,49.49,56.31,115.26,76.71,276.72,0.0,0.0,0.0,0.0
thunderstorm with light rain,0.0,0.0,5.82,0.0,20.32,0,25.73,0.0,205.29,0.0,0.0,5.82,0.0,0.0,0.0,0.0,20.32,0,28.24,0.0,0.0,0.0,0.0,296.94,0.0,38.42,127.85,0.0,0.0,0.0,0.0,0.0
thunderstorm with heavy rain,0.0,0.0,0,0.0,0,0,0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0.0,0,0,0,0.0,0.0,0.0,0.0,0,0.0,0,383.54,0.0,0.0,0.0,0.0,0.0
proximity thunderstorm with rain,0.0,0.0,1.94,0.0,0.0,0,0,0.0,68.43,0.0,0.0,1.94,0.0,0.0,0.0,0.0,0.0,0,0,0.0,18.01,0.0,20.0,0.0,112.61,115.26,0.0,0.0,0.0,0.0,0.0,0.0
heavy intensity drizzle,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
proximity shower rain,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
freezing rain,0.0,0.0,0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
shower snow,0.0,0.0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
