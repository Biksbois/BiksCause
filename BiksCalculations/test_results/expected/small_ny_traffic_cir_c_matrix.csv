,Clouds,Clear,Mist,Snow,Haze,Rain,Drizzle,Fog,Thunderstorm,overcast clouds,sky is clear,mist,broken clouds,heavy snow,few clouds,scattered clouds,haze,light rain,light intensity drizzle,fog,moderate rain,snow,drizzle,proximity thunderstorm,heavy intensity rain,thunderstorm with light rain,proximity thunderstorm with rain,thunderstorm with heavy rain,proximity shower rain,shower snow,freezing rain,heavy intensity drizzle
Clouds,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Clear,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Mist,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Snow,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Haze,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Drizzle,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Fog,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
Thunderstorm,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
overcast clouds,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
sky is clear,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
mist,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
broken clouds,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
heavy snow,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
few clouds,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
scattered clouds,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
haze,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
light rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
light intensity drizzle,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
fog,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
moderate rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
snow,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
drizzle,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
proximity thunderstorm,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
heavy intensity rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
thunderstorm with light rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
proximity thunderstorm with rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
thunderstorm with heavy rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
proximity shower rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
shower snow,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
freezing rain,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
heavy intensity drizzle,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0
